src/main/java/bio/terra/service/snapshotbuilder/utils/AggregateBQQueryResultsUtils.java

```diff
@@ -13,9 +14,11 @@ public static List<SnapshotBuilderConcept> aggregateConceptResults(TableResult r
             row -> {
               int count;
               try {
-                count = (int) row.get("count").getLongValue(); // If exists, use its value
+                count =
+                    SnapshotBuilderService.fuzzyLowCount(
+                        (int) row.get("count").getLongValue()); // If exists, use its value
               } catch (IllegalArgumentException e) {
-                count = 1;
+                count = SnapshotBuilderService.fuzzyLowCount(1);
```

Comment:
Since this is stub data that will go away in a future PR there's no need to "fuzz" it. Maybe change it to `0` if you don't want to return `1` from the API.